database/migrations/2019_06_03_214715_create_jobs_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateJobsTable extends Migration
```

Comment:
This should probably be called `CreateQueuedJobsTable` for the sake of avoiding confusion.